Build.PL

```diff
@@ -100,6 +100,7 @@ EOW
       "test_requires" => {
         "File::Spec" => 0,
         "Module::Metadata" => 0,
+        "Mojolicious" => "7.49",
```

Comment:
This needs to be added in prereqs.yml not here.

Reply:
Actually please just bump the Mojolicious runtime dep - there's no practical reason for test to have a separate dep.